### ISSUE TITLE
Incorrect "Unused LANGUAGE pragma" warning

### ISSUE THREAD

**Issue by TMVector** [ndmitchell/hlint#540]
I get a warning despite needing `DeriveAnyClass` to derive `NFData`:
```
Test.hs:1:1: Warning: Unused LANGUAGE pragma
Found:
  {-# LANGUAGE DeriveAnyClass #-}
Why not remove it.
```

```haskell
{-# LANGUAGE DeriveAnyClass     #-}
{-# LANGUAGE DeriveGeneric      #-}
{-# LANGUAGE StandaloneDeriving #-}

{-# ANN module "HLint: ignore Unused LANGUAGE pragma" #-}
module Test where

data TestFrame a = TestFrame
  { frameValue :: a
  , frameDone  :: Bool
  }
  deriving (Eq, Generic, Show)

deriving instance NFData a => NFData (TestFrame a)
```

**Comment by ndmitchell** [comment]
Thanks for the report. I just tried it and it works for me, giving no hints. Which version of HLint are you using? Can you try upgrading?

**Comment by TMVector** [comment]
Ah, apologies -- I am using 2.1 and it seems to be fixed by at least 2.1.6. Thanks, and sorry to have wasted your time!

**Comment by ndmitchell** [comment]
No problem, I appreciate the bug report!
